molecule/default/tests/test_default.py

```diff
@@ -56,10 +57,18 @@ def test_authorized_keys(host):
 
 def test_inactive_user(host):
     passwd = host.file('/etc/passwd')
-    sudoers = host.file('/etc/sudoers.d/inactiveuser')
+    group = host.file('/etc/group')
 
     assert not passwd.contains(inactiveuser)
-    assert not sudoers.exists
+    assert not group.contains(inactiveuser)
+
+
+def test_removed_user(host):
+    passwd = host.file('/etc/passwd')
+    group = host.file('/etc/group')
+
+    assert not passwd.contains(removeduser)
+    assert not group.contains(inactiveuser)
```

Comment:
Typo? Pretty sure the removeduser should no longer be in /etc/group either.
```suggestion
    assert not group.contains(removeduser)
```